src/IncrementalChecker.ts

```diff
@@ -128,9 +133,29 @@ class IncrementalChecker {
     return endsWith(filePath, '.d.ts') || linterExclusions.some(matcher => matcher.match(filePath));
   }
 
+  /**
+   * Check for vue-loader in user's project node_modules.
+   */
+  hasVueLoader(): boolean {
+    if (this.vueLoaderResolved !== undefined) return this.vueLoaderResolved;
+    
+    try {
+      Resolve.sync('vue-loader', { basedir: this.basedir });
```

Comment:
do we need `Resolve` to test if vue-loader is installed? Maybe we could use built-in node functionality?
```
try {
    require.resolve("vue-loader");
    this.vueLoaderResolved = true;
} catch (e) {
    this.vueLoaderResolved = false;
}
```